activerecord/lib/active_record/core.rb

```diff
@@ -295,6 +295,7 @@ def arel_table # :nodoc:
       def arel_attribute(name, table = arel_table) # :nodoc:
         table[name]
       end
+      deprecate :arel_attribute
```

Comment:
What's the deprecation message in here? Does it suggest to use `arel_table` directly?

Reply:
I'd not like to suggest using private API.

Reply:
Ahh, that makes sense. Thanks for info.